frontend/src/modules/Traces/TraceGraph.tsx

```diff
@@ -69,25 +84,37 @@ const _TraceGraph = (props: TraceGraphProps) => {
 	useEffect(() => {
 		if (props.traceItem) {
 			let sortedData = getSortedData([tree]);
```

Comment:
we should pass as tree only as it will be array

Reply:
`tree` is an object here. `getSortedData` expects an array